recipes/kallisto/meta.yaml

```diff
@@ -7,20 +7,23 @@ package:
 source:
   fn: kallisto-{{ version }}.tar.gz
   url: https://github.com/pachterlab/kallisto/archive/v{{ version }}.tar.gz
-  sha256: 2164938c2c61c04e338c4c132cf749f56d39e6f0b4c517121bca1fbc218e430e
+  sha256: 35a81201a56f4557697e6fe693dc6b701bbbd0a7b2b6e1c6c845ef816d67ca29
+  patches:
+    - cmakelists.patch
+    - cmakelists-src.patch
 
 build:
     number: 0
     string: hdf5{{CONDA_HDF5}}_{{PKG_BUILDNUM}}
 
 requirements:
     build:
-        - hdf5 {{CONDA_HDF5}}
+        - hdf5 {{ CONDA_HDF5 }}*
         - cmake
         - zlib
```

Comment:
zlib line should also be updated so it is pinned

Reply:
Good point! Curious this didn't came up during linting.

Reply:
Me we fixed this recently ... lets try again.

Reply:
Oh wait, you don't have it as run-time dependency, so this is a static build right? If so you don't need to pin it.